homeassistant/components/rki_covid/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "rki_covid",
+  "iot_class": "cloud_polling",
+  "name": "Robert-Koch Institut COVID Numbers",
+  "documentation": "https://www.home-assistant.io/integrations/rki_covid",
+  "dependencies": [],
+  "codeowners": ["@thebino"],
+  "requirements": ["rki-covid-parser==1.3.1"],
+  "quality_scale": "gold",
```

Comment:
I am not sure if you can list `gold` already. Have a look at https://developers.home-assistant.io/docs/integration_quality_scale_index/#gold-.
For example, I miss 
> Tests for fetching data from the integration and controlling it

Reply:
There is now a 91% coverage. I think this box is checked now as well again.